templates/user_profiles/email_add_user.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base_email.html" %}
+{% block email_content %}
+<p>Bonjour,</p>
```

Comment:
comme cet email va à tous les controleurs, on pourrait specifier qui a fait l'ajout? ("Bli Bla vient d'ajouter une nouvelle personne")

Sinon les destinataires en cc vont voir "vous venez d'ajouter une personne" alors qu'ils ont rien fait, yen a qui vont psychoter. (je sais pas si tout les destinataires seront tres au courant de la difference entre "to" et "cc")

Du coup, pour "penser a l'inviter", faudrait mettre plutot "Bli Bla, pensez a l'inviter" (pour qu'une seule personne l'invite)

Reply:
ok